tests/integration/targets/docker_image/tasks/tests/docker_image.yml

```diff
@@ -136,6 +136,10 @@
     - repository_1 is changed
     - repository_2 is not changed
 
+- assert:
+    that:
+      - 'FROM busybox' in repository_1.stdout
```

Comment:
Can you comment these three lines out? I'll uncomment them when porting this change to community.docker.